src/components/Tagline/Tagline.scss

```diff
@@ -0,0 +1,9 @@
+@import '~@quartz/styles/scss/color-scheme';
+@import '~@quartz/styles/scss/fonts';
+
+.container {
+	@include font-maison-500-1;
+
+	color: $color-typography-faint;
+	line-height: 1;
```

Comment:
Is it intentional not to use the line-height from the mixin here? Should it be updated in the mixin instead?

Reply:
Good question. I'll let the mixin prevail and we can see how it shakes out in QA.